bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/MagnitudeSystem.scala

```diff
@@ -1,14 +1,25 @@
 package edu.gemini.spModel.core
 
+import scala.collection.JavaConversions._
 import scalaz.Equal
 
-sealed abstract class MagnitudeSystem(val name: String) extends Product with Serializable
+sealed abstract class MagnitudeSystem(val name: String) extends Product with Comparable[MagnitudeSystem] with Serializable {
+
+  // ===== LEGACY JAVA SUPPORT =====
+  import MagnitudeSystem._
+
+  def compareTo(other: MagnitudeSystem) = all.indexOf(other) - all.indexOf(this)
```

Comment:
this is kind of strange, is there a natural way to order `MagnitudeSystems`? Is it done anywhere?